prime-angular-frontend/src/app/modules/site-registration/pages/overview/overview.component.ts

```diff
@@ -97,4 +110,27 @@ export class OverviewComponent implements OnInit {
       this.showSubmission = !this.site.submittedDate;
     }
   }
+
+  /**
+   * @description
+   * Infer whether user was required to accept an Organization Agreement.
+   * User saw an Organization Agreement if the current Site has a care setting that is unique
+   * for that Organization.
+   */
+  private determineOrgAgreementRequired$(): Observable<SiteListViewModel[]> {
+    const currentCareSettingCode: number = this.site.careSettingCode;
+
+    return this.siteResource.getSites(this.organization.id).pipe(
+      tap(sites => {
+        sites.forEach(
+          site => {
+            this.logger.trace("doingBusinessAs", site.doingBusinessAs);
```

Comment:
This will only log in local, but i'm not sure if we want to leave these in as they don't seem super relevant outside debugging this PR.

Reply:
Addressed in latest commit 68fb52e